axlearn/common/checkpointer.py

```diff
@@ -315,11 +315,14 @@ class Config(StateStorage.Config):
                 `None` and `1` means no sharding. `-1` means fully shard along data-parallel
                 replicas. `>1` means custom sharding degree (currently not implemented).
             max_concurrent_gb: Max concurrent shards (in GB) to write.
+            max_concurrent_restore_gb: Max concurrent shards (in GB) to read during checkpoint
+                restore. `None` or `0` means using a default value of 32GB.
```

Comment:
Why do we use 32GB as the default? Is it for backwards compatibility?

Reply:
Yeah for backwards compatibility.